assets/home/tasks-section/ready.js

```diff
@@ -33,6 +34,22 @@ const Ready = ( { onDismiss } ) => {
 		} );
 	};
 
+	// Prepare social media links.
+	const shareLink = encodeURIComponent( coursePermalink );
+	const facebookLink = `https://www.facebook.com/sharer/sharer.php?u=${ shareLink }`;
+	const tumblerLink = sprintf(
+		// translators: %1$s placeholder is the share link, %2$s placeholder is the share link.
```

Comment:
I think the URL by itself comment isn't translatable, so in theory, we shouldn't need this comment here, no?

Reply:
Good catch! https://github.com/Automattic/sensei/pull/5946/commits/8509d91174f46f133382834c00338da313d3ad5e